### ISSUE TITLE
Build instructions/APK download

### ISSUE THREAD

**Issue by ThatOneCalculator** [johnyob/beacon#3]
Build instructions, and APK download, or even an F-droid listing if possible would be great for people who want to use this app!

**Comment by mxbi** [comment]
Hi there, thanks a lot for the interest! In terms of building it, it shouldn't be too hard just building it as a standard flutter app, we didn't do anything unusual (sorry I don't remember exactly I don't have it installed currently!)

**I've attached the APK here in case you want to try it out on your own devices**, but a bit of a health warning:
- This was a hackathon project, and so while it works on our devices (and should continue to because there are no servers!) it's not production ready, and we do not provide any warranty.
- In particular, the E2EE implementation should NOT be relied on to be uncrackable (we wrote our own crypto as a learning challenge), and we have not done any security hardening.

[Beacon_base.apk.zip](https://github.com/beacon-meshenger/beacon/files/7793724/Beacon_base.apk.zip)

If you try it out, let us know if it works! 😊 (or if you run into any issues)
